src/pages/tutorials/smart-rollups/optimize.md

```diff
@@ -0,0 +1,68 @@
+---
+id: optimize
+title: "Part 4: Optimizing the kernel"
+lastUpdated: 11th October 2023
+---
+
+The kernel must be efficient with space and processing power because there is a size limit for kernels.
+In these steps, you optimize the kernel:
+
+1. Run this command to print the current size of the kernel:
+
+   ```bash
+   du -h target/wasm32-unknown-unknown/debug/hello_world_kernel.wasm
+   ```
+
+   You can run this command inside or outside of the Docker container.
+
+   The size of the compiled kernel and its dependencies may be 18MB or more, which is too large to deploy.
```

Comment:
This is made much worse by us using a `debug_build`. It's actually `1.9MB` in a `release` build. To be honest, that should probably be the first thing to try?

Reply:
Added a release build with the commands `cargo build --release --target wasm32-unknown-unknown` and `du -h target/wasm32-unknown-unknown/release/hello_world_kernel.wasm`.